kadi/commands/tests/test_commands.py

```diff
@@ -314,6 +316,32 @@ def test_commands_create_archive_regress(tmpdir, version_env):
                 # Path("cmds_local.txt").write_text(out)
                 assert len(cmds_flight) == len(cmds_local)
 
+            # Roughly validate NSM quaternions and remove from later comparison
```

Comment:
Comment should reflect *why* the code is here. The *what* it is doing is also useful but that is fairly evident in the code.